source/src_pw/hamilt_cu.bak

```diff
@@ -2,7 +2,7 @@
 #include "global.h"
 #include "hamilt.h"
 #include "diago_cg.h"
-#include "diago_cg_gpu.h" // Diago_CG_GPU
+#include "diago_cg.h" // Diago_CG_CUDA
```

Comment:
What is the usage of this file: hamilt_cu.bak ?

Reply:
This file has been deleted.